modules/core/include/opencv2/core/eigen.hpp

```diff
@@ -72,6 +72,14 @@ namespace cv
 #ifdef OPENCV_EIGEN_TENSOR_SUPPORT
 /** @brief Converts an Eigen::Tensor to a cv::Mat.
 
+ These functions are provided for OpenCV-Eigen interoperability. They convert `Mat`
```

Comment:
Wrong place.
It must be a group documentation instead of single function's note under `#ifdef`

Reply:
Still wrong.
http://pullrequest.opencv.org/buildbot/export/pr/17368/docs/d0/daf/group__core__eigen.html